Update "Set up web development for Flutter" page (#11362)
flutter/website: src/_includes/docs/install/reqs/add-web.md

Download and install [Google Chrome][] to run and debug Flutter web apps.

> Alternatively you can install [Microsoft Edge][] on Windows, or use a different

Reviewer: ```suggestion
Alternatively, you can install [Microsoft Edge][] on Windows, or use a different
```

web browser by using the `flutter run -d web-server` command and navigating to

> the specified URL, but debugging is in this mode may be limited.

Reviewer: ```suggestion
the specified URL; however, debugging is in this mode might be limited.
```
